Assignment2/hxshfx-fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,143 @@
+{
+    "@context":{
+
+        "schema":"http://schema.org/",
+        "rdfs":"http://www.w3.org/2000/01/rdf-schema#",
+        "foaf":"http://xmlns.com/foaf/0.1/",
+        "dbp":"http://dbpedia.org/resource/classes#",
```

Comment:
If you want to reuse the DBpedia ontology the correct URI is http://dbpedia.org/ontology/
You don't need to change it in your assignment, but for the future you already know.